barretenberg/cpp/scripts/ci_benchmark_ivc_flows.sh

```diff
@@ -85,6 +85,10 @@ function chonk_flow {
   echo "$flow ($runtime) has proven in $((elapsed_ms / 1000))s and peak memory of ${memory_taken_mb}MB."
   dump_fail "verify_ivc_flow $flow $output/proof"
   echo "$flow ($runtime) has verified."
+  # Get proof size after compression
+  tar -czf "$output/proof.tar.gz" -C "$output" proof
```

Comment:
should use snappy compress not tar in context of p2p

Reply:
Did quick test, snappy is about 2% worse